packages/office-addin-manifest/src/manifestHandler/manifestHandlerJson.ts

```diff
@@ -37,21 +37,21 @@ export class ManifestHandlerJson extends ManifestHandler {
     }
   }
 
-  getManifestInfo(teamsAppManifest: TeamsAppManifest): ManifestInfo {
+  getManifestInfo(teamsAppManifest: any): ManifestInfo {
```

Comment:
Why are we changing this to "any"?  Typescript discourages the use of any suggesting that "unknown" is better . . . but a strong type is even more desirable.

Reply:
This should be temporary. `TeamsAppManifest` is not updated to include some of the fields here.
So I put any while it is not update to get latest schema changes.

Reply:
You should leave a comment to that effect and log a bug to fix it.

Reply:
Sure, added a TODO comment and created a bug to fix it